system/Validation/Rules.php

```diff
@@ -427,7 +424,10 @@ public function required_without($str = null, string $fields, array $data): bool
 		// any of the fields are not present in $data
 		foreach ($fields as $field)
 		{
-			if (! array_key_exists($field, $data))
+			if (
+				(strpos($field, '.') !== false && empty(dot_array_search($field, $data))) ||
+				(strpos($field, '.') === false && (! array_key_exists($field, $data) || empty($data[$field])))
```

Comment:
Same thing here. Please reverse the order.

Reply:
Done!